ISSUE_TEMPLATE.md

```diff
@@ -1,88 +1,8 @@
-        - **cc `@kubernetes/feature-reviewers` on this issue** to get approval before checking this off
-        - New apis: *Glossary Section Item* in the docs repo: kubernetes/kubernetes.github.io
-      - [ ] Update release notes
-- [ ] Beta
-  - [ ] Testing is sufficient for beta
-  - [ ] User docs with tutorials
-        - *Updated walkthrough / tutorial* in the docs repo: kubernetes/kubernetes.github.io
-        - cc `@kubernetes/docs` on docs PR
-        - **cc `@kubernetes/feature-reviewers` on this issue** to get approval before checking this off
-  - [ ] Thorough API review
-    - **cc `@kubernetes/api`**
-- [ ] Stable
-  - [ ] docs/proposals/foo.md moved to docs/design/foo.md 
-        - **cc `@kubernetes/feature-reviewers` on this issue** to get approval before checking this off
-  - [ ] Soak, load testing 			
-  - [ ] detailed user docs and examples
-    - **cc `@kubernetes/docs`**
-    - **cc `@kubernetes/feature-reviewers` on this issue** to get approval before checking this off
-
-*FEATURE_STATUS is used for feature tracking and to be updated by `@kubernetes/feature-reviewers`.*
-**FEATURE_STATUS: IN_DEVELOPMENT**
-
-More advice:
-
-Design
-   - Once you get LGTM from a *`@kubernetes/feature-reviewers`* member, you can check this checkbox, and the reviewer will apply the "design-complete" label.
- 
-Coding
-  - Use as many PRs as you need.  Write tests in the same or different PRs, as is convenient for you.
-  - As each PR is merged, add a comment to this issue referencing the PRs.  Code goes in the http://github.com/kubernetes/kubernetes repository,
-        and sometimes http://github.com/kubernetes/contrib, or other repos.
-  - When you are done with the code, apply the "code-complete" label.
-  - When the feature has user docs, please add a comment mentioning `@kubernetes/feature-reviewers` and they will
-        check that the code matches the proposed feature and design, and that everything is done, and that there is adequate
-        testing.  They won't do detailed code review: that already happened when your PRs were reviewed.
-        When that is done, you can check this box and the reviewer will apply the "code-complete" label.
-
-Docs
-  - [ ] Write user docs and get them merged in.
-  - User docs go into http://github.com/kubernetes/kubernetes.github.io.
-  - When the feature has user docs, please add a comment mentioning `@kubernetes/docs`.
-  - When you get LGTM, you can check this checkbox, and the reviewer will apply the "docs-complete" label.
-
+# Feature Description
+- One-line feature description (can be used as a release note);
+- Primary contact (assignee);
+- Responsible SIG;
+- Design proposal link (community repo);
+- Reviewer(s) - (for LGTM) recommend having 2+ reviewers (at least one from code-area OWNERS file) agreed to review. Reviewers from multiple companies preferred;
+- Approver (likely from SIG/area to which feature belongs);
+- Initial target stage (alpha/beta/stable) and release (x.y);
```

Comment:
Will all new features be expected to go through the alpha/beta/stable release cycle? For example, adding a new flag to kubectl is a relatively minor addition that probably doesn't need additional, long-term testing. 

If that's the case, should such a feature be submitted through this workflow? I don't see an explanation as to when a contributor should submit a feature request here or through a normal feature PR. The only reference is in the README which says it's provisional.